website/docs/components/app-side-nav/partials/code/how-to-use.md

```diff
@@ -0,0 +1,387 @@
+<Hds::AppSideNav>
+  ...
+  <:footer>
+    <OrganizationPicker
+      @currentOrg={{this.currentOrg}}
+      @organizations={{this.organizations}}
+      {{! apply this special class to automatically fade in/out }}
+      class='hds-app-side-nav-hide-when-minimized'
+    />
+  </:footer>
+</Hds::AppSideNav>
+```
+
+Alternatively, if you want to create a custom transition (or respond in a different way to the minimization of the width) you can use the variable `isMinimized` provided by the `<:footer>` named block:
+
+```handlebars{data-execute=false}
+<Hds::AppSideNav>
+  ...
+  <:footer as |Footer|>
+
+    {{! show/hide when minimization is toggled }}
+    {{#unless Footer.isMinimized}}
+      [ YOUR CUSTOM CONTENT HERE ]
+    {{/unless}}
+
+    {{! apply custom class (eg. to transition) when minimization is toggled }}
+    <div class={{if Footer.isMinimized "is-minimized" "is-expanded"}}>
+      [ YOUR CUSTOM CONTENT HERE ]
+    <div>
+  </:footer>
+</Hds::AppSideNav>
+```
+
+### Responsiveness
+
+As mentioned above, the full-fledged `Hds::AppSideNav` component is “responsive” by default:
+
+- when the **viewport is `desktop`**, the sidebar navigation is static and has a fixed width
+    - the width at which the viewport is considered “desktop” is controlled by a dedicated CSS variable `--hds-app-desktop-breakpoint`; if needed it can be overwritten (at `:root` level) to define a custom “desktop” breakpoint
+- when the **viewport is `mobile`**, the sidebar navigation is responsive and the user can minimize or maximize its width via a toggle button (the component will take care automatically of the transitions between these two states)
+    - in this case the App Side Nav occupies only a minimum width in terms of page layout, even when expanded (it partially covers the main content)
+    - a toggle button is added to the App Side Nav, used to expand/minimize its width
+    - an overlay is automatically added to the main content area so that the user can’t interact with it while the App Side Nav is opened (if the user clicks on the overlay, the App Side Nav closes and returns to its minimized state; the same happens if the user presses the `esc` key)
+
+This “responsive” behavior can be programmatically turned off passing a value `false` to the argument `@isResponsive` in the `Hds::AppSideNav` component.
+
+_Notice that even if the `@isResponsive` parameter is set to `false`, some JavaScript is executed anyway in the background, and events listeners are attached to some DOM elements (even if these functionality are not used)._
+
+#### Logical and visual states
+
+When the component is “responsive” there are different “states” (and associated logic) for different combinations of these three parameters:
+
+- `isResponsive`
+- `isDesktop`
+- `isMinimized`
+
+![Schema of the different visual states in response to the change of viewports and the corresponding logic variables](/assets/components/app-side-nav/responsiveness.png)
```

Comment:
Nit: I think it would be better to try and get more of this information out of the image and into the page. Maybe a table with the images and captions describing each state?